Makefile

```diff
@@ -83,7 +83,18 @@ manifests:
 
 generate: $(KUSTOMIZE) go.generate manifests.annotate
 
-
+CROSSPLANE_CHART_DIR=cluster/charts/crossplane
+CRD_DIR = $(CROSSPLANE_CHART_DIR)/templates/crds
+HELM3_CRD_DIR = $(CROSSPLANE_CHART_DIR)/crds
+manifests.annotate:
```

Comment:
nitpic about spacing and flow..

The comment block that was above manifests.annotate should stay above the `manifests.annotate:` variables.  Your new comments should be incorporated into that comment block.

https://github.com/crossplaneio/crossplane/pull/1077/files#diff-b67911656ef5d18c4ae36cb6741b7965R86 should match the spacing of the other two variables.

Reply:
Thanks, both should be done now.